specification/intelligentdataexperiences/resource-manager/Microsoft.AadCustomSecurityDiagnosticSettings/preview/2023-03-01-preview/intelligentdataexperiences.json

```diff
@@ -0,0 +1,486 @@
+{
+  "swagger": "2.0",
```

Comment:
A few questions regarding your Swagger Spec PR:
1. Looks like you are onboarding your service using non-RPaaS RP. All the new services need to be onboarded using RPaaS. If you have got exception already for going with non-RPaaS RP, please confirm. Otherwise you need to discuss about exception here: [https://aka.ms/RPaaSException](https://aka.ms/RPaaSException). See more details here: [https://armwiki.azurewebsites.net/rp_onboarding/process/onboarding.html](https://armwiki.azurewebsites.net/rp_onboarding/process/onboarding.html)).

2. Is the RP Namespace Microsoft.AadCustomSecurityDiagnosticSettings already reviewed and approved from Naming Consul. See here:
[https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#resource-provider-and-type-names-best-practices](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#resource-provider-and-type-names-best-practices)
[https://microsoft.sharepoint.com/sites/globalreadiness/SitePages/NewNameReview.aspx](https://microsoft.sharepoint.com/sites/globalreadiness/SitePages/NewNameReview.aspx)

3. Did you meet anyone in ARM to explain your scenario and get first review?

Reply:
Thank you @betrigg for providing answers to these questions. One follow-up question: I understand "Custom Security Attributes" name is internally discussed and approved, however RP namespace has to be reviewed with Naming Consul (Branding team, Legal team etc.) to align with public aspects. Please reach out to Naming Consul as well: https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#resource-provider-and-type-names-best-practices.